lib/netns_linux.c

```diff
@@ -590,3 +592,64 @@ ns_id_t ns_get_default_id(void)
 		return default_ns->ns_id;
 	return NS_DEFAULT_INTERNAL;
 }
+
+bool netns_get_name(char *name, int len)
+{
+	char net_path[PATH_MAX];
+	int netns;
+	struct stat netst;
+	DIR *dir;
+	struct dirent *entry;
+
+	name[0] = '\0';
+
+	snprintf(net_path, sizeof(net_path), "/proc/%d/ns/net", getpid());
```

Comment:
we should keep #ifdef HAVE_NETNS

Reply:
I really don't understand what you are trying to say here.  this is netns_linux.c so we are guaranteed to have HAVE_NETNS defined.

Reply:
I propose you to clean code with 'HAVE_NETNS' currently present on netns_linux.c file.